third_party/Microsoft/Dump/MPEG2Dump.cpp

```diff
@@ -1282,7 +1282,7 @@ HRESULT CMPEG2Dump::WriteNetwork(PBYTE data, DWORD len)
 		temp[--pos] = (TCHAR) digit+L'0';
 	} while (li.QuadPart);
 	sprintf(request, "WRITE %s %d\r\n", temp+pos, m_bRegOptimizeTransfers ? m_bytesLeftInNetworkWrite : len);
-	int dataSize = strlen(request);
+	size_t dataSize = strlen(request);

```

Comment:
Did this generate a compiler warning for the next line regarding a signed vs. unsigned comparison?

Reply:
On review send requires an int and size of request is 512
Reverted to int also @ 1680 and 1704
Added comments